### PR TITLE
Update GDScript template

### PR DESCRIPTION
**Pull request by razcore-rad**
The GDScript template always bugged me because of the start + end white spaces.... yah, very important, I know :P

### CODE REVIEW

File: modules/gdscript/gd_editor.cpp [hunk 1]
```diff
@@ -47,16 +47,14 @@ void GDScriptLanguage::get_string_delimiters(List<String> *p_delimiters) const {
...
+	"# class member variables go here, for example:\n"+
+	"# var a = 2\n"+
+	"# var b = \"textvar\"\n\n"+
 	"func _ready():\n"+
 	"\t# Called every time the node is added to the scene.\n"+
 	"\t# Initialization here\n"+
-	"\tpass\n"+
-	"\n"+
-	"\n";
+	"\tpass";
```
Comment (akien-mga) [inline modules/gdscript/gd_editor.cpp]: Please keep just one final newline, for us Unix users it's very important :D

Comment (neikeq) [inline modules/gdscript/gd_editor.cpp]: Missing `\n` at the end of the file.

Comment (razvanc-r) [inline modules/gdscript/gd_editor.cpp]: I hate that end newline but fine I updated :P

**Comment by akien-mga** [comment]
Could you squash the commits? ;)

**Comment by razvanc-r** [comment]
OK, I'm not sure what I did but hopefully I did the squash D:

**Pull request merged**